pattern-library/sass/components/_icons.scss

```diff
@@ -10,5 +10,362 @@
 .icon {
     display: inline-block;
     font-size: 1em;
-    fill: currentColor;
+    color: currentColor;
```

Comment:
Cool, so this value (``currentColor``) won't work then, right? If not, can you adjust it so that it inherits the color of the parent?